modules/cpmso-yaml-provider-spec-bare-metal.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control-plane-machine-management.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-provider-spec-bare-metal_{context}"]
+= Sample bare metal provider specification
```

Comment:
This section is included as a stub to show my planned framework, and will be commented out for merge, then completed in a later PR.

Placeholder content is AWS.

Reply:
Ok cool - I no longer recall where I saw bare metal mentioned but easy enough to omit :)